tools/data-handler/src/containers/project.ts

```diff
@@ -233,6 +233,34 @@ export class Project extends CardContainer {
     return resources;
   }
 
+  /**
+   * Returns a new unique card key with project prefix (e.g. test_x649it4x).
+   * @returns a new card-key string
+   */
```

Comment:
Add to the comment blocks that this throws if it cannot generate a valid key after maxAttempts.

Reply:
Fixed in [852fc33](https://github.com/CyberismoCom/cyberismo/pull/127/commits/852fc3324963ad17dac30dd7baa1091d6b0f5c39)